src/app.ts

```diff
@@ -0,0 +1,95 @@
+﻿let grid: iq.onboard.Grid,
+	refreshHandle: number = 99;
+
+$(() => {
+	grid = new iq.onboard.Grid("#grid");
+
+	initialize();
+});
+
+function initialize() {
```

Comment:
If you define a function outside a module it will pollute the global namespace, and can cause some issues if someone else uses the same naming (third party libraries are the biggest risk)